src/routes/Landing.tsx

```diff
@@ -1,18 +1,60 @@
 import React, { Fragment } from 'react';
+import { useFlag } from '@unleash/proxy-client-react';
 
 import '../components/app-content-renderer/styles/panels.scss';
 
 import FirstPanel from '../components/app-content-renderer/first-panel';
 import SecondPanel from '../components/app-content-renderer/second-panel';
 import VirtualAssistant from '../components/app-content-renderer/virtual-assistant';
+import { useLoadModule } from '@scalprum/react-core';
+import { PageSection } from '@patternfly/react-core/dist/dynamic/components/Page';
+
+const getWidgetLayoutLandingPage = () => {
+  const scope = 'widgetLayout';
```

Comment:
I know the API is not doing this yet because we had no widgets but I just want to make sure this was said. Scope and module should be custom attributes of the layout. The widgets module should be "stupid" and not know which widgets exist or not. Technically it should be just a simple render loop and the `ScalprumComponent` should handle the rest.

One generic component for every widget type should be sufficient.

Reply:
That should be part of the widget layout. This is just loading the widget layout into landing page.

Reply:
I know. As I stated,  just want to make sure this was said.